node/tasks.py

```diff
@@ -80,7 +80,10 @@ def send_app_txs(app_name: str) -> None:
         )
     except Exception:
         zlogger.exception("An unexpected error occurred:")
-        zdb.add_missed_txs(app_name=app_name, txs=initialized_txs)
+        if len(initialized_txs) != 0:
+            zdb.add_missed_txs(app_name=app_name, txs=initialized_txs)
+            return
```

Comment:
Should this `return` not be done after setting `is_sequencer_down` to `true`?
or moving `zdb.is_sequencer_down = True` to above `if`?

Reply:
Fixed in this [commit](https://github.com/zellular-xyz/zsequencer/commit/ee5f0e7ebd51bb68f3c8d9b84186f11d1a58df64)